engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -95,7 +95,9 @@
 public class StramClient
 {
   private static final Logger LOG = LoggerFactory.getLogger(StramClient.class);
-  public static final String YARN_APPLICATION_TYPE = "DataTorrent";
+  public static final String YARN_APPLICATION_TYPE_DEPRECATED = "DataTorrent";
+  public static final String YARN_APPLICATION_TYPE = "ApacheApex";
```

Comment:
Let us stick with "ApacheApex", as
1. we need unique name for the Application type.
2. All the packages are(will) using org.apache.apex.. should we change them as well?
3. If at all the we have to change for a reason you mentioned then we have bigger things to worry than the type of an application